README.rst

```diff
@@ -29,6 +29,16 @@ platform specific code.
 .. contents:: **Contents**
 
 
+Development at GitHub.com
+-----------------------------
+
+Development of wxPython is done with git, using
+https://github.com/wxWidgets/Phoenix as the main devlopment site.
+
+Currently (September, 2024), only 2 developers (@swt2c, @RobinD42) can merge
+pull requests, or assign others to review pull requests.
```

Comment:
That last part isn't right.  Anyone can review pull requests without being assigned.

Reply:
I think it is right.  The statement is about who can assign other to review.   I think GH refers to this as "who can Manage and Triage" issues and pull requests.   These roles do not have to be an owner.    In many large projects, there are multiple people assigning reviewers, and all PRs need a review before merging.

Reply:
OK so you mean specifically assigning reviewers, not reviewing itself.  OK.